src/package-index.html

```diff
@@ -8,10 +8,12 @@
     <link href="https://fonts.googleapis.com/icon?family=Material+Icons+Outlined" rel="stylesheet" />
   </head>
   <body class="tce-m-0 dark:tce-bg-slate-800 dark:tce-text-slate-50">
-    <noscript
-      >Javascript appears to be disabled - unfortunately the estimator requires it to be enabled to function.</noscript
-    >
-    <tech-carbon-estimator></tech-carbon-estimator>
-    <footer class="tce-p-5 tce-bg-slate-800 tce-text-slate-50 dark:tce-bg-slate-950">© Copyright Scott Logic</footer>
+    <main>
```

Comment:
Should this `main` element have the same padding as `src/index.html`?

Reply:
I think it's fine for this one to stay as it is? This one is just really for testing any issues that might come up when using it on the TCS site, without having to make a package and update it.